src/cloud/release-notes/ece-release-notes.md

```diff
@@ -17,9 +17,30 @@ The `{{site.data.var.ct}}` package uses the following release versioning sequenc
 {:.bs-callout-info}
 See [Upgrades and patches]({{ site.baseurl }}/cloud/project/project-upgrade-parent.html) for information about updating to the latest release of the `{{site.data.var.ct}}` package.
 
-## v2002.1.4
+## v2002.1.5
 *Release date: {{ site.data.var.ece-release-date }}*<br/>
 
+-  {:.new}**Remote storage**—Added the [`REMOTE_STORAGE`]({{site.baseurl}}/cloud/env/variables-deploy.html#remote_storage) environment variable to enable Cloud Projects for remote storage of media files using a storage service such as AWS S3.<!--MCLOUD-7153-->
+
+-  {:.new}**Flushing the opcache**—Added support for the `opcache.enable_cli` PHP option to flush the opcache before running the deploy hook. This configuration resets the cache configuration to ensure that the current configuration settings are applied on each deployment.<!--MCLOUD-7015-->
```

Comment:
Looks like you are using the proper name here; if so, it may need to be caps. I'm not sure. I would expect all lowercase to be in code format, or to be an adjective before the word cache. Your choice.

```suggestion
-  {:.new}**Flushing the opcache**—Added support for the `opcache.enable_cli` PHP option to flush the OPcache before running the deploy hook. This configuration resets the cache configuration to ensure that the current configuration settings are applied on each deployment.<!--MCLOUD-7015-->
```